src/basics/Icons/stories.tsx

```diff
@@ -19,20 +19,39 @@ Storybook.storiesOf('basics|Icons', module).add('index', () => {
 const compose = (...functions: ((arg: any) => any)[]) => (arg: any) =>
     functions.reduceRight((p, f) => f(p), arg)
 
-const toSvgRaw = (iconMap: { [key: string]: string }) => (key: string) =>
-    iconMap[key]
+const toSvgRawWithName = (iconMap: { [key: string]: string }) => (
+    key: string
+) => ({
+    html: iconMap[key],
+    name: key
+})
 
-const renderIcon = (fill?: string) => (html: string): React.ReactElement => (
-    <Icon dangerouslySetInnerHTML={{ __html: html }} fill={fill} />
+const renderIcon = (fill?: string) => (arg: {
```

Comment:
IMO: 
個人的にparamsって定義すること多いんですけど、どっちでもいいです

https://blog.masahiko.info/entry/2008/05/28/181539
```suggestion
const renderIcon = (fill?: string) => (params: {
```